pages/osx/readlink.md

```diff
@@ -0,0 +1,8 @@
+# readlink
+
+> Follow symlinks and get symlink information.
+> More information: <https://www.gnu.org/software/coreutils/readlink>.
+
+- Print the actual full path to which the symlink points:
+
+`readlink {{filename}}`
```

Comment:
I'd maybe say `path/to/symlink` as well, wdyt?
```suggestion
`readlink {{path/to/symlink}}`
```

Reply:
good idea